kite-tools-parent/kite-tools/src/main/java/org/kitesdk/cli/commands/DeleteCommand.java

```diff
@@ -31,6 +32,10 @@
   @Parameter(description = "<dataset or view>")
   List<String> targets;
 
+  @Parameter(names="--dryRun",
```

Comment:
The convention for other arguments is lower case separated by dash. Could you change this to `--dry-run`?